app/assets/stylesheets/base/colorswatches.scss

```diff
@@ -1,10 +1,13 @@
-$primary-color-light-2: #fbeed0;
-$primary-color-light-1: #f4cd71;
-$primary-color: #e3a412;
-$primary-color-dark-1: #8e670b;
-$primary-color-dark-2: #2f2204;
+$primary-color-light-2: #B6C7F0;
+$primary-color-light-1: #85A0DD;
+$primary-color: #5F7EC6;
+$primary-color-dark-1: #4061AD;
+$primary-color-dark-2: #264A9C;
 
-$secondary-color: #627BB3;
+$secondary-color-light-2: #ACEDED;
+$secondary-color: #4EBABA;
+$secondary-color-dark-1: #2E9C9C;
+$secondary-color-dark-2: #168C8C;
 
-$error: red;
-$success: green;
\ No newline at end of file
+$error: grey;
```

Comment:
@aSadhankar I think errors should ultimately still be some kind of red color. If you want to change the where this is used, I'd maybe create a new color called `$info: grey;` and then use that in the rest of the CSS.